snmp/apache-stats.py

```diff
@@ -18,72 +18,68 @@
 #
 #
 
-CACHETIME = 30
-CACHEFILE = '/var/cache/librenms/apache-snmp'
+CACHETIME=30
```

Comment:
it's python not bash
please do space between operators
-> PEP8 compatibilty

Reply:
Sorry, will do.